your-delivery/src/main/java/com/ch/yourdelivery/owner/respository/OwnerRepositoryImpl.java

```diff
@@ -0,0 +1,32 @@
+package com.ch.yourdelivery.owner.respository;
+
+import com.ch.yourdelivery.owner.domain.model.Owner;
+import org.springframework.stereotype.Repository;
+
+import java.util.HashMap;
+
+@Repository
+public class OwnerRepositoryImpl implements OwnerRepository {
+
+    final HashMap<String, Owner> ownerTable = new HashMap<String, Owner>();
```

Comment:
생성할때 제네릭 타입을 넣지않아도 됩니다
```suggestion
    final HashMap<String, Owner> ownerTable = new HashMap<>();
```

Reply:
넵 알려주셔서 감사합니다!

Reply:
test:)

Reply:
test2